src/libs/auth.js

```diff
@@ -52,8 +87,7 @@ const becameRegistered = (oldState, state) => {
  */
 export const ensureBillingScope = async () => {
   if (!hasBillingScope()) {
-    const options = new window.gapi.auth2.SigninOptionsBuilder({ scope: 'https://www.googleapis.com/auth/cloud-billing' })
-    await getAuthInstance().currentUser.get().grant(options)
+    await signIn(true)
     // Wait 250ms before continuing to avoid errors due to delays in applying the new scope grant
```

Comment:
I wonder if this delay is still needed given the different API being used.

Reply:
it is not!